QUARKUS-2489.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-2489 OIDC Back channel logout
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2489
+
+This feature provides Back-Channel Logout capability for OIDC extension.
+
+## Scope of the testing
+
+Backchannel logout applies to `quarkus-oidc` and basically is a standard way to logout the current user from all the applications this user is currently logged in, bypassing the user agent.
+
+We will create a valid token by an authorization flow and proceed with the logout flow.
+### Impact on test suites and testing automation
+
+A New logout endpoint is going to be created in `keycloak-oidc-client-extended` and `keycloak-oidc-client-reactive-extended`, and we will need to cover JVM and native mode in order to double-check that there are no conflicts with the native compilations. 
+### Impact on resources:
+
+The new scenario is going to run in JVM and native mode over baremetal and OpenShift. So we are expecting about 5 min per module on JVM mode, and 10 min per module on native mode. Running a test over OpenShift used to take some extra time but is just an aproximation. To sum up, we are expecting an increase of 30 min of total amount execution.
+## Getting familiar with the feature
+
+Following actions were taken to ensure familiarity:
+- Ensure documentation provides a clear explanation of expected behavior
+- Review upstream and quickstarts coverage
+- Read openID connect specifications
```

Comment:
Please rewrite to _openID connect_ to _OpenID Connect_.

Reply:
@pjgg Changes look good, but what about openID connect, I requested changes as all linked references use OpenID Connect. If you find request for change unreasonable, please explain why. Thank you